### ISSUE TITLE
Make static click option in relative mode

### ISSUE THREAD

**Issue by Lgreenbunny** [Lgreenbunny/randomAutoHotkeys#21]
If there's a static click in the queue, it should always click in that coordinate based on the Screen CoordMode. When adding relative coordinates after a static click, the distance after the last relative click/anchor point should be logged instead of the static click(s)'s, and when going through the queue, it should calculate a screen coordinate with the last relative coordinate's screen equivalent and the distance logged before to continue the queue in relative mode.
Basically, it should perform like a regular relative mode click sequence, but with a static click somewhere. After the static click, the relative clicks shouldn't be effected by it.
